deploy/terraform/variables.tf

```diff
@@ -542,6 +559,141 @@ variable "vpc_cidr_block" {
   default     = "10.0.0.0/16"
 }
 
+variable "allow_all_cidr_block" {
+  description = "CIDR block to allow all IP addresses"
+  type = string
+  default = "0.0.0.0/0"
+}
+
+variable "allow_all_ipv6_cidr_block" {
+  description = "IPV6 CIDR block to allow all IP addresses"
+  type = string
+  default = "::/0"
+}
+
+variable "tcp_network_protocol" {
+  description = "tcp protocol for VPC"
+  type = string
+  default = "TCP"
+}
+
+variable "allow_all_vpc_protocol" {
+  description = "Allow all protocol for VPC"
+  type = string
+  default = "-1"
+}
+
+variable "allow_all_vpc_port" {
+  description = "Allow all port for VPC"
+  type = number
+  default = 0
+}
+
+variable "mq_engine_type" {
```

Comment:
Some of these "variables" feel less configurable than others.

Being able to control the `mq_engine_version` makes sense to me, but being able to switch out what engine type feels a lot less configurable 🤔.